templates/site/node.html

```diff
@@ -14,20 +16,24 @@
     {% include "partials/banner.html" %}
 {% endwith %}
 
-{% with %}
-    {% set title = "Start a Project" %}
-    {% set subtitle = "Select most suitable node or server to start a project" %}
-
-    {% include "partials/general-title-subtitle.html" %}
-{% endwith %}
+    <div class="container-fluid d-none d-lg-block mt-5 w-100">
```

Comment:
This only shows on lg screen? I see that's what it shows in Figma but are we sure? @ShrutiK100

Reply:
I just realized while reviewing that I haven't added the heading "Start the project" and body under it. I have added it in the Figma design now